main/components/__init__.py

```diff
@@ -58,15 +70,49 @@ def remove(self, id: int) -> None:
         </ul>
         {{message}}
     </div>
-    """
+    
+<script>
+document.addEventListener('DOMContentLoaded', function() {
+  const componentContainer = document.querySelector('[tetra-component="main__default__person_form_component"]');
 
-    def form_valid(self, form) -> None:
-        instance = form.save(commit=False)
-        instance.save()
-        self.message = "Person successfully saved."
+  function attachFormListener() {
+    const form = document.getElementById('personForm');
+    form.addEventListener('submit', function(event) {
+      event.preventDefault();
+      const formData = new FormData(form);
 
-    def form_invalid(self, form) -> None:
-        self.message = "Error saving person."
+      fetch(form.action, {
```

Comment:
While this may work in theory, it is besides all mechanics of tetra itself. Tetra does the form upload in `tetra.core.js`, and uses Alpine.morph to morph the response into the content. This may not be circumvented by just doing things "manually". It should be included in the tetra upload workflow, ideally in `tetra.core.js`.